tests/search/reindexing/reindexing.rb

```diff
@@ -0,0 +1,122 @@
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+require 'indexed_search_test'
+
+class ReindexingTest < IndexedSearchTest
+
+  DOCUMENT_COUNT = 1000
+  MUSIC_CLUSTER_ID = 'music'
+  MOVIE_CLUSTER_ID = 'movie'
+  MUSIC_DOC_TYPE = 'music'
+  MOVIE_DOC_TYPE = 'movie'
+
+  def setup
+    set_owner('bjorncs')
+  end
+
+  def test_reindexing_with_multiple_content_clusters
+    app = SearchApp.new.monitoring('vespa', 60).
+        cluster(SearchCluster.new(MUSIC_CLUSTER_ID).sd(selfdir + 'music.sd')).
+        cluster(SearchCluster.new(MOVIE_CLUSTER_ID).sd(selfdir + 'movie.sd')).
+        container(Container.new('combinedcontainer').
+            jvmargs('-Xms16g -Xmx16g').
```

Comment:
Possibly a bit high for a non-performance-test. 
```suggestion
```

Reply:
Fixed :)